src/main/java/heronarts/lx/color/LXPalette.java

```diff
@@ -505,15 +505,28 @@ private void _transitionColor(LXDynamicColor color, int i, float lerp) {
     case HSV:
```

Comment:
Not your fault, but I do wish this naming was consistent with BlendMode (HSV2). IMO, "HSV" should be shortest path in both locations, and HSVCW or HSVCCW could do the other modes that might take a path over 180 degrees.

Reply:
Erf, yeah this is a bit of a mess for sure. The CW/CCW suggestion isn't a bad idea. Unfortunately may bork saved project files doing this change, but maybe worth just biting the bullet and eating that cost now rather than later. Will mull on this.